n2y/export.py

```diff
@@ -58,11 +62,21 @@ def export_page(
     property_map=None,
 ):
     pandoc_ast = page.to_pandoc()
+    page_properties = _page_properties(
+        page, pandoc_format, id_property, url_property, property_map,
+    )
+    if pandoc_format.startswith("docx"):
+        output_path = Path(tempfile.mkdtemp()) / "page.docx"
+        pandoc_write_or_log_errors(pandoc_ast, pandoc_format, pandoc_options, path=output_path)
+        if not output_path.exists():
+            logger.warning("An empty page will produce corrupt docx output (%r)", page.notion_url)
+        if page_properties:
+            with OpenOfficeMetadata(output_path, "a") as oof:
```

Comment:
Why is all of this code necessary?

I thought the "other-pandoc-formats" branch was already at a point where pandoc would insert the page properties into the word document metadata. Thus, I would have expected that the only changes needed to accomplish https://www.notion.so/medtech-os/Make-swap-header-footer-script-get-meta-data-from-word-files-bef0b0bf025147c19800310402c64791?pvs=4 would be in the notion-qms repository.

Reply:
For the simple-database-to-files test, the only metadata generated are the database fields Name and Tags. Of those, Tags is falsy.